insights/parsr/iniparser.py

```diff
@@ -1,6 +1,6 @@
 import string
 
-from insights.parsr import (EOF, HangingString, InSet, LeftBracket, Lift,
+from insights.parsr import (Comma, EOF, EOL, DoubleQuotedString, HangingString, InSet, LeftBracket, Lift,
```

Comment:
@shlao - would you please adjust the length of these lines to make it a bit alignment and not exceed the 75? Thanks.